### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by MingjiaLiu1995**

### CODE REVIEW

**Comment by MingjiaLiu1995** [comment]
Hi @wangyun-microsoft, could you help check whether the information in the change log is proper? We'll replace the links with aka when the content freezes. Please feel free to add review suggestions if any comment.

@hermanwenhe Still need some time to lock down the content before pr merge

File: CHANGELOG.md
Comment (wangyun-microsoft) [inline CHANGELOG.md:1]: line 13: Word JavaScript API 1.9
line 14: Word JavaScript API desktop-only requirement set 1.1

Do we want to add this link as well? https://devblogs.microsoft.com/microsoft365dev/whats-new-for-office-add-ins-at-ignite-2024/

**Comment by wangyun-microsoft** [comment]
I've added a comment.

line 13: Word JavaScript API 1.9
line 14: Word JavaScript API desktop-only requirement set 1.1
Do we want to add this link as well? https://devblogs.microsoft.com/microsoft365dev/whats-new-for-office-add-ins-at-ignite-2024/


Thanks.



Wang Yun (王云)

Product Manager, Office Platform

***@***.***?anonymous&ep=bwmEmailSignature>            Book time to meet with ***@***.***?anonymous&ep=bwmEmailSignature>



________________________________

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+## 1.0.0 - Dec 13, 2024
```
Comment (hermanwenhe) [inline CHANGELOG.md:1]:
```suggestion
## 1.0.0 - Dec 19, 2024
```
Update release date

**Comment by MingjiaLiu1995** [comment]
Updated the change log content according to comments @hermanwenhe @wangyun-microsoft

**Review by hermanwenhe — APPROVED** [review]

**Pull request merged**